spot-contracts/contracts/_interfaces/IPerpetualTranche.sol

```diff
@@ -79,6 +75,15 @@ interface IPerpetualTranche is IERC20Upgradeable {
     /// @return Address of the deposit bond.
     function getDepositBond() external returns (IBondController);
 
+    /// @notice The tranche token contract currently accepted to mint perp tokens.
+    /// @return Address of the deposit tranche ERC-20 token.
+    function getDepositTranche() external returns (ITranche);
+
+    /// @notice Returns the ratio of tranches held in perp and the vault.
```

Comment:
I think this returns the tranche ratios as configured in the bond, not the ratio of tranches held in perp and the vault, per se?

If so, we should make that clear in the function name. Maybe something like `getBondSeniorJuniorRatios`?

Reply:
Well we use this terminology through out. In the fee policy for example. 
perpTR -> Tranche ratio of seniors accepted by perp
vaultTR -> The remainder tranche ratio held in the vault

There are three ratios at play here. Not sure whats the best way to name them. 
1) The senior/junior tranche ratio 
2) The ratio of value in perp / vault
3) The "target" ratio of value in perp/vault

Maybe here we qualify these are tranche ratios ie) `getPerpVaultTrancheRatios`?

Reply:
I'd recommend--
1) If we're talking about "tranche" ratios, that means as configured in the bond. These are descriptive properties of the tranche tokens / bond
2) If we're talking about "value" or "tvl" ratios, that means the ratio of values in the two parts of the system
3) Then "target" only ever applies to "target value ratio"

So we should probably never have "vault" and "tranche" as qualifiers at the same time.
We should also never have "target" and "tranche" as qualifiers at the same time.

seniorTR and juniorTR combine to form tranchRatio
perpTVL and vaultTVL and trancheRatio combine to form subscriptionRatio
subscriptionRatio and targetSubscriptionRatio combine to form normalizedSR

Something like that?

Reply:
That makes the most sense. Go to the source, and that self-documents